src/components/member-card/index.js

```diff
@@ -7,16 +7,32 @@ import PropTypes from 'prop-types';
 import ShowSkills from '@components/member-card/show-skills';
 import SuperUserOptions from '@components/member-card/super-user-options';
 
-const Card = ({ developerInfo, isOptionKey }) => {
+const Card = ({ developerInfo, isOptionKey, colorCombination }) => {
   const { query } = useRouter() || { query: { dev: false } };
   const { dev } = query;
   const { username, first_name, last_name, img_url, isMember } = developerInfo;
+  const { color_primary, color_secondary } = colorCombination;
+  const cardRef = useRef();
+  const textRef = useRef();
+
   const socialMedia = [
     'twitter_id',
     'github_id',
     'linkedin_id',
     'instagram_id',
   ];
+
+  useEffect(() => {
+    const cardElement = cardRef.current;
+    const textElement = textRef.current;
+    if (cardElement) {
+      cardElement.style.color = color_primary;
+      cardElement.style.border = `2px solid ${color_primary}`;
+      cardElement.style.backgroundColor = color_secondary;
+      textElement.innerText = `${first_name[0].toUpperCase()} ${last_name[0].toUpperCase()}`;
```

Comment:
Can the styling come from CSS modifiers instead?
We add the modifying class for the color(s), and the CSS included does the rest? Will make our code a lot more declarative in nature, and reduce the number of lines here.

Reply:
for now i have added inline styles, later might make the suggested change, once I get the idea about the hashing function